### ISSUE TITLE
추가 데이터 사용 여부

### ISSUE THREAD

**Issue by hagg30** [naver/nlp-challenge#8]
NER과 SRL 태스크에서 주어진 학습 코퍼스 이외에 다른 공개된 추가 데이터(공개 코퍼스, 공개된 pre-trained model)를 사용해도 되는지 확인 부탁드립니다.

**Comment by kduk** [comment]
네, 사용하셔도 됩니다. 다만 NSML 시스템 및 정책 상으로 허용 가능한 범위(e.g. 모델 포함 총 사이즈 5GB 이내)를 한정하고 있습니다. 아래 내용을 확인 부탁드립니다. 외부 데이터를 NSML에서 활용하기 위한 가이드 및 데이터 사이즈 관련 내용은 다시 공유드리겠습니다.

아래와 같은 가이드를 FAQ에 추가했습니다.
https://github.com/naver/nlp-challenge/blob/master/FAQ.md

-----------------------------

### Q. 공개된 데이터 이외에 추가 데이터를 사용해도 될까요?
가능합니다. 예를 들어 pre-train된 embedding model을 활용한다든지, 한국어 wikipedia 사전이나 다른 사전등을 활용하셔도 됩니다.
단, 다음 조건을 만족해야 합니다.
  - 외부에 공개된 데이터이며, 누구나 접근 가능한 데이터여야 한다.
  - 사용된 외부 데이터의 종류 및 접근 방법(원소스)는 추후 공개되어야 한다.

외부 리소스를 NSML에서 활용하는 방법에 대한 가이드는 준비 중에 있습니다. 준비되면 공유드리겠습니다.

**Comment by rkdls** [comment]
방법은 크게 2가지가 있습니다.

1. 코드상에서 특정위치의 데이터/pretrained model을 다운받고 training/ model load 하는 부분을 임의로 만드셔서 생성된 모델을 ```nsml.save(model name)``` 함수를 사용해서 모델을 저장하시면 됩니다. (코드내에서 다 작성하셔야합니다.)

2. NSML 에서는 코드를 실행할때 도커환경에서 실행되는데요, 원하는 pretrained model, data가 저장된 docker image를 직접 빌드하셔서 docker hub에 푸시해서 사용하시거나, 빌드되어있는 docker image를 NSML 의 ```setup.py, requirements.txt``` 에 기입하여서 사용하는 방법이 있습니다.(https://n-clair.github.io/nlp-challenge-docs/_build/html/ko_KR/contents/session/prepare_a_session_environment.html#)

감사합니다.
